apps/console/src/features/applications/components/settings/attribute-management/attribute-selection.tsx

```diff
@@ -719,14 +744,13 @@ export const AttributeSelection: FunctionComponent<AttributeSelectionPropsInterf
                                                                             claimMappingOn={ claimMappingOn }
                                                                             claimMappingError={ claimMappingError }
                                                                             readOnly={
-                                                                                (selectedSubjectValue === claim.claimURI)
+                                                                                (selectedSubjectValue ===
```

Comment:
Let's format this line properly

Reply:
Addressed with https://github.com/wso2/identity-apps/pull/2136/commits/ee4252ee9ebc4c53695c6349e0daed00b03ebf15